articles/getting_to_know/howto/graphics/camera/HowTo_Understand_Different_Camera_Modes.md

```diff
@@ -0,0 +1,287 @@
+---
+title: Understanding different camera modes
+description: Demonstrates Several camera views in a single script.
+requireMSLicense: true
+---
+
+## Overview
+
+This example is different from most as it is demonstrating several techniques to update a camera's view and position to achieve different camera modes or operations, including:
+
+- A Fixed Camera.
+- A Fixed tracking camera - follows the target.
+- A first person camera.
+- A third person camera - with spring.
+- A top-down fixed camera view.
+- A top-down view centered on a target.
+
+### End result
+
+These are the kinds of camera modes demonstrated in this sample:
+
+|Fixed View|Fixed Tracking|First Person View|
+|-|-|-|
+|![Fixed](../images/HowTo_CameraModes_Fixed.gif)|![Fixed Tracking](../images/HowTo_CameraModes_Tracking.gif)|![First Person](../images/HowTo_CameraModes_FirstPerson.gif)|
+|Third Person View|Top Down|Top Down tracking|
+|![Third Person](../images/HowTo_CameraModes_ThirdPerson.gif)|![Top-Down](../images/HowTo_CameraModes_TopDownFixed.gif)|![Top Down tracking](../images/HowTo_CameraModes_TopDownTracking.gif)|
+
+## Sample
+
+As the sample is quite large and contains several pieces of content for the demonstration, it is provided as a download here:
+
+### [Basic Camera Example](../files/BasicCameraExample.zip)
```

Comment:
maybe a Tutorials repo, which has the code and a github actions job which zips up the projects into their own zip files and publishes them?

Reply:
It is a fair point, I didn't "like it" but we do not really have a place to store document tutorials.
Maybe I'll setup a new repo for sample code, but not projects/demos.

Or should we just create another area on the MonoGame.Samples repo, rather than just crating more repos?

Reply:
I'm ok with it being in Monogame.Samples. just needs to be in their own sub directory for Tutorials. 
We can setup a build job to create the source zip files for each tutorial (same for the samples as well I guess).

Reply:
OK, I'll migrate the code for the tutorial to the Samples in it's own folder.

Reply:
Sample uploaded to the Samples repo, once merged, I'll update the links in this doc and remove the zip @dellis1972 
https://github.com/MonoGame/MonoGame.Samples/pull/80